ipfx/nwb_reader.py

```diff
@@ -90,6 +90,9 @@ def read_sweep_from_source(source):
 
         raise ValueError("Could not find a source/sweep_number attribute/dataset.")
 
+    def get_starting_time(self, data_set_name):
+        NotImplementedError
```

Comment:
Should this be `raise NotImplementedError`?

Reply:
@njmei Good catch, thanks